domain/src/main/java/com/example/domain/usecase/document/DocumentUseCaseImpl.kt

```diff
@@ -54,8 +53,10 @@ class DocumentUseCaseImpl @Inject constructor(
     override fun addFavoriteTutor(tutorId: String): Flow<Either<ExceptionState, Boolean>> = flow {
         val response = tutorDataSource.addFavoriteTutor(tutorId)
 
+        val failedResponseCode = 1.0
+
         val dataConvert = response.mapAndConverterToStateData {
-            it.result != 1.0
+            it.result != failedResponseCode
```

Comment:
Call check from constant value

Reply:
because const value can't use in local variables and I think this value is fit with this api so i only create local variables for this case